Mathlib/Analysis/Normed/Ring/Basic.lean

```diff
@@ -195,9 +201,6 @@ section NonUnitalSeminormedRing
 
 variable [NonUnitalSeminormedRing α] {a a₁ a₂ b c : α}
 
-theorem norm_mul_le (a b : α) : ‖a * b‖ ≤ ‖a‖ * ‖b‖ :=
-  NonUnitalSeminormedRing.norm_mul _ _
```

Comment:
I'd be tempted to leave this in place, so that `nnnorm_mul_le` and `norm_mul_le` are in the same namespace. You can `protected` the field to prevent issues with clashes.

Reply:
>  So is there really a benefit in providing a root-level lemma that repeats the definition?

We do this in a handful of other places. Not doing it leads to things like `exact?`, `show_term` etc including the namespace, which is usually not what we want.

Reply:
I think this is especially confusing since the `norm_mul_le` field is duplicated between typeclasses, and so to a casual reader its not necessarily obvious which one is the preferred one. (in the past I tried to switch things to use `extends`, but I ran out of time and gave up)